charts/alfresco-activiti/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+description: A Helm chart for Alfresco Activiti
+name: alfresco-activiti
```

Comment:
are those the best names?

I see that on docs https://docs.alfresco.com/process-services/latest/install/manual/

we are calling them process services and process services administrator

Reply:
names validated with PM Arabella. @vprovaggi was in the chat but did not reply yet